# This config file is used by ros2_control
controller_manager:
  ros__parameters:
    update_rate: 31  # Hz

    rebel_arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController


    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

rebel_arm_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5
      - joint6
    gains:
      joint1:
        ff_velocity_scale: 1.0
      joint2:
        ff_velocity_scale: 1.0
      joint3:
        ff_velocity_scale: 1.0
      joint4:
        ff_velocity_scale: 1.0
      joint5:
        ff_velocity_scale: 1.0
      joint6:
        ff_velocity_scale: 1.0
    command_interfaces:
      - velocity
    state_interfaces:
      - position
      - velocity